aten/src/ATen/autocast_mode.cpp

```diff
@@ -406,6 +406,7 @@ TORCH_LIBRARY_IMPL(aten, AutocastCPU, m) {
   KERNEL_CPU2(conv_transpose2d, input, lower_precision_fp)
   KERNEL_CPU2(conv_transpose3d, input, lower_precision_fp)
   KERNEL_CPU(prelu, lower_precision_fp)
+  KERNEL_CPU(scaled_dot_product_attention, lower_precision_fp)
```

Comment:
this is needed because slow path for mha eventually calls into sdpa [over here](https://github.com/pytorch/pytorch/blob/546e5358020620b688a47f1cfce0b3804b826b60/torch/nn/functional.py#L5441)